README.md

```diff
@@ -1,38 +1,51 @@
+| `RULES_DIR`        | Comma-separated list of directories to walk in order to source rules files                                                                                                                                                                 | `false`  | `./`    |
 
 ## Actions
 
+All actions will crawl the specified `RULES_DIR` for Prometheus rules and alerts files with a `yaml`/`yml` extension.
+
 ### `diff`
 
-Running the action with the `diff` command will crawl the specified `RULES_DIR` for Prometheus rules files with a `yaml`/`yml` extension and output the differences in the configured files and the currently configured ruleset in a Cortex cluster. It will output the required operations in order to make the running Cortex cluster match the rules configured in the directory. It will **not create/update/delete any rules** currently running in the Cortex cluster.
+Outputs the differences in the configured files and the currently configured ruleset in a Cortex cluster. It will output the required operations in order to make the running Cortex cluster match the rules configured in the directory. It will **not create/update/delete any rules** currently running in the Cortex cluster.
 
 ### `sync`
 
-Running the action with the `diff` command will crawl the specified `RULES_DIR` for `yaml`/`yml` Prometheus rules files and reconcile the differences with the sourced rules and the rules currently running in a configured Cortex cluster. It **will create/update/delete rules** currently running in Cortex to match what is configured in the files in the provided directory.
+Reconcile the differences with the sourced rules and the rules currently running in a configured Cortex cluster. It **will create/update/delete rules** currently running in Cortex to match what is configured in the files in the provided directory.
+
+### `lint`
+
+Lints a rules file(s). The linter's aim is not to verify correctness but just YAML and PromQL expression formatting within the rule file(s). The linting happens in-place within the specified file(s). Does not interact with your Cortex cluster.
+
+### `prepare`
+Prepares a rules file(s) for upload to Cortex. It lints all your PromQL expressions and adds a `cluster` label to your PromQL query aggregations in the file. Prepare modifies the file(s) in-place. Does not interact with your Cortex cluster.
+
+### `check`
+
+Checks rules fuile(s) against the recommended [best practices](https://prometheus.io/docs/practices/rules/) for rules. Does not interact with your Cortex cluster.
 
 ## Outputs
 
 ### `summary`
 
-The `summary` output variable returned by this action is a string denoting the number of rule groups either created/updated/deleted. If the action is set to `diff`, these values are only hypothetical.
+The `summary` output variable is a string denoting the output summary of the action, if there is one.
```

Comment:
might be useful to have examples here and for detailed, like should the summary just be the exit code and detailed be the log lines? this would also give people an idea of what to set these values to when they write new commands within the action

Reply:
At the moment, they are not consistent across all the commands. I think we should first tackle the consistency of the outputs before we even attempt to document them in some form. #1 briefly touches this subject, but I have added another comment to make sure we document them as well.